examples/petsc/bps.c

```diff
@@ -762,6 +762,7 @@ int main(int argc, char **argv) {
     my_rt_start = MPI_Wtime();
     ierr = KSPSolve(ksp, rhs, X); CHKERRQ(ierr);
     my_rt = MPI_Wtime() - my_rt_start;
+    MPI_Allreduce(&my_rt, &my_rt, 1, MPI_DOUBLE, MPI_MIN, comm);
```

Comment:
Oops, should have been.
```suggestion
    MPI_Allreduce(MPI_IN_PLACE, &my_rt, 1, MPI_DOUBLE, MPI_MIN, comm);
```